compiler+runtime/src/cpp/jank/read/parse.cpp

```diff
@@ -348,6 +348,41 @@ namespace jank::read::parse
 
     if(character.is_none())
     {
+      /* Unicode */
+      if(sv[0] == '\\' && sv[1] == 'u')
+      {
+        /* Should always be of length 6 eg. \uaBc5 */
```

Comment:
This is how Clojure handles it, but I don't think we should have this limitation. For example, `\u1f920` is needed for 🤠. Same goes for octal. We should read however much is there, I think, as long as it's one codepoint.